app/components/wallet/hwConnect/common/CheckDialog.scss

```diff
@@ -46,17 +48,14 @@
     }
 
     li {
-      margin-left: 18px;
       margin-top: 12px;
-      font-size: 16px;
```

Comment:
probably this will affect a lot of screens 🤔

Reply:
@nicarq 
It's part of cleanup, as many redundancy was there.
Before-Classic Theme:
`font-size: 16px;`

After-Classic Theme:
`font-size: 16px;`

Before-Modern Theme:
`font-size: 14px;`

After-Modern Theme:
`font-size: 14px;`